MediaBrowser.MediaEncoding/Subtitles/SubtitleEncoder.cs

```diff
@@ -559,6 +637,15 @@ private async Task ExtractAllExtractableSubtitlesInternal(
                     outputPath);
             }
 
+            await ExtractSubtitlesForFile(inputPath, args, outputPaths, cancellationToken).ConfigureAwait(false);
```

Comment:
I think we should exit if `outputPaths.Count == 0` - MKSs only subs.

Reply:
Agreed. I added an early abort there.